e2e/tests/notebook/notebook-create-python.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('Python Kernel Notebook Test', () => {
 
     // Verify kernel name.
     const kernelName = await notebook.getKernelName();
-    expect(kernelName).toBe(JupyterKernels.Python);
+    expect(kernelName).toContain(JupyterKernels.Python);
```

Comment:
Now matches either `Python 3` or `Python 3 (ipykernel)`.  I don't think this test needs to care that it matches exactly.